utilities/markChoreDone.js

```diff
@@ -1,26 +1,84 @@
+//node packages
+const {google} = require("googleapis");
+require("dotenv").config();
+
 //local packages
 const {
-  app: {
-    client: {
-      chat: { update }
+    app: {
+        client: {
+            chat: { update }
+        }
     }
-  }
 } = require("./bolt.js");
+const privatekey = require("../keys/sheets-api.json");
 
 //globals
 const TOKEN = process.env.SLACK_BOT_TOKEN;
+const SPREADSHEET_ID = process.env.SPREADSHEET_ID;
+const GDRIVE_EMAIL = process.env.GDRIVE_EMAIL;
+
+const jwtClient = new google.auth.JWT(
+    privatekey.client_email,
+    null,
+    privatekey.private_key,
+    ["https://www.googleapis.com/auth/spreadsheets"],
+    GDRIVE_EMAIL
+);
+
+const markChoreUser = async (user, date, choretext) => {
+    const sheets = google.sheets({
+        version: "v4",
+        auth: jwtClient
+    });
+
+    jwtClient.authorize(err => console.log(err ? err : "Successfully connected!"));
+
+    try {
+        const {data} = await sheets.spreadsheets.values.get({
+            spreadsheetId: SPREADSHEET_ID,
+            range: "A2:B"
+        });
+        for (let n = 0; n < data.values.length; n++) {
```

Comment:
Can we do `for (n of data.values)`? Maybe change `n` to something more meaningful?

Reply:
Never mind me. I didn't realize you were using the int `n` as a value in your code.